### ISSUE TITLE
was this works for 10.14 or 10.13 high sierra or mojave ?

### ISSUE THREAD

**Issue by developerss** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#145]
i asked because i have failed to install high sierra or mojave in 10700 processor , 
have it success in high sierra or mojave ?

**Comment by Got99** [comment]
no



发自我的小米手机
在 Asif ***@***.***>，2021年3月15日 下午10:18写道：

i asked because i have failed to install high sierra or mojave in 10700 processor ,
have it success in high sierra or mojave ?

**Comment by developerss** [comment]
how to reply to that email , i have done unsubscribed

**Comment by MiddlemanTM** [comment]
Yes it can do. I have successfully got it working on High Sierra and Mojave, Catalina and Big Sur on i5-10500.
You need to spoof the Coffee Lake CPUID (in reverse byte order) in the Kernel section.
![cpuid-070421](https://user-images.githubusercontent.com/78093030/113813017-a93fed80-97a1-11eb-9f8b-944e0cd73ff5.png)
![Screen Shot 2021-02-11 B](https://user-images.githubusercontent.com/78093030/113812251-587bc500-97a0-11eb-809a-74c16b075443.png)

**Comment by developerss** [comment]
omg i cant believe please share the oc forlder or clover folder

**Comment by developerss** [comment]
how you changed processor name ?

> Yes it can do. I have successfully got it working on High Sierra and Mojave, Catalina and Big Sur on i5-10500.
> You need to spoof the Coffee Lake CPUID (in reverse byte order) in the Kernel section.
> ![cpuid-070421](https://user-images.githubusercontent.com/78093030/113813017-a93fed80-97a1-11eb-9f8b-944e0cd73ff5.png)
> ![Screen Shot 2021-02-11 B](https://user-images.githubusercontent.com/78093030/113812251-587bc500-97a0-11eb-809a-74c16b075443.png)

omg i cant believe please share the oc forlder or clover folder

**Comment by developerss** [comment]
awesome brother greate workssss , awesome , if possible share the software which you have changed the processor name

**Comment by developerss** [comment]
my gtx 1650 i cant use on cataline . but high sierra can use it , awesome works ,

**Comment by developerss** [comment]
is just change cpuid ? cpuid max too ?

**Comment by developerss** [comment]
> Yes it can do. I have successfully got it working on High Sierra and Mojave, Catalina and Big Sur on i5-10500.
> You need to spoof the Coffee Lake CPUID (in reverse byte order) in the Kernel section.
> ![cpuid-070421](https://user-images.githubusercontent.com/78093030/113813017-a93fed80-97a1-11eb-9f8b-944e0cd73ff5.png)
> ![Screen Shot 2021-02-11 B](https://user-images.githubusercontent.com/78093030/113812251-587bc500-97a0-11eb-809a-74c16b075443.png)

brother you are the best the best the best ever , 
really only thanks not enough ,
many mnany many many lot lot lotl ottlottttttttt thanksssssssss

**Comment by developerss** [comment]
> Yes it can do. I have successfully got it working on High Sierra and Mojave, Catalina and Big Sur on i5-10500.
> You need to spoof the Coffee Lake CPUID (in reverse byte order) in the Kernel section.
> ![cpuid-070421](https://user-images.githubusercontent.com/78093030/113813017-a93fed80-97a1-11eb-9f8b-944e0cd73ff5.png)
> ![Screen Shot 2021-02-11 B](https://user-images.githubusercontent.com/78093030/113812251-587bc500-97a0-11eb-809a-74c16b075443.png)

i get error msg in last two miniute remaining - installtion process
"An error occurred while preparing the installation."

**Comment by developerss** [comment]
you used appfs format or hfs ? to install it

**Comment by developerss** [comment]
i installed offline dmg works like charm

**Comment by developerss** [comment]
> Yes it can do. I have successfully got it working on High Sierra and Mojave, Catalina and Big Sur on i5-10500.
> You need to spoof the Coffee Lake CPUID (in reverse byte order) in the Kernel section.
> ![cpuid-070421](https://user-images.githubusercontent.com/78093030/113813017-a93fed80-97a1-11eb-9f8b-944e0cd73ff5.png)
> ![Screen Shot 2021-02-11 B](https://user-images.githubusercontent.com/78093030/113812251-587bc500-97a0-11eb-809a-74c16b075443.png)

thanks again , thanks again
you just make me so happy
